src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -78,6 +80,17 @@ internal struct ParsedStat
             //internal long cguest_time;
         }
 
+        internal struct ParsedStatus
+        {
+            internal int Pid;
```

Comment:
Can we delete this field?

Reply:
Can't. Need it for asserts.

E.g. ` Debug.Assert(!b || result.Pid == pid, "Expected process ID from status file to match supplied pid");`

Reply:
Asserts are only used in debug.  If you want to keep the asserts. you can #if DEBUG around the asserts, the field, and the parsing of the field; then you get to keep the asserts but we don't pay for parsing the field and carrying it around when it's not used for anything in release.